juntagrico/entity/jobs.py

```diff
@@ -16,14 +16,17 @@ class ActivityArea(JuntagricoBaseModel):
     name = models.CharField(_('Name'), max_length=100, unique=True)
     description = models.TextField(
         _('Beschreibung'), max_length=1000, default='')
-    email = models.EmailField(null=True, blank=True)
     core = models.BooleanField(_('Kernbereich'), default=False)
-    hidden = models.BooleanField(_('versteckt'), default=False)
-    coordinator = models.ForeignKey('Member', on_delete=models.PROTECT)
-    members = models.ManyToManyField(
-        'Member', related_name='areas', blank=True)
+    hidden = models.BooleanField(
+        _('versteckt'), default=False,
+        help_text=_('Nicht auf der "Kernbereiche"-Seite anzeigen. Einsätze bleiben sichtbar.'))
```

Comment:
Falsch auf der tätigkeits bereichs seite wird es nicht angezeigt
sollte visible for members oder so heissen und der beschrieb genau das "Sichtbar für Mitglieder" 
des weiteren ist es wahscheinlich dann auch so, dass man die jobs nicht anzeigen sollte

Reply:
Wir nutzen die Funktion aktuell so, dass auf der Seite der Tätigkeitsbereiche nur jene Bereiche angezeigt werden, die auch von einer aktiven AG gestützt sind, damit es da übersichtlich bleibt. Für andere Bereiche werden aber auch noch aktuelle Jobs ausgeschrieben. Daher würde ich die Funktion nicht auf die Sichtbarkeit der Jobs ausweiten.

Reply:
also die ursprüngliche idee für das feature war, dass man tätigkeits bereiche hat, für die sich die members nicht selber einschreiben konnten. zum beispiel buchhaltung weil man da ja gewisses know how haben muss. auch die jobs die da stattfinden sind dann eigentlich nur für die mitglieder des tätigkeitsbereichs und werden wahrscheinlich ers nacher erstellt und befüllt. von daher wäre es gut wenn die nirgends auftauchen.

Reply:
eventuell sollte man das visible for member machen welches genau das macht und ein members can registrate welches es auf der tätigkeitsseite zwar nicht anzeigt oder besser nur den slider wegnimmt und die jobs aber sichtbar sind